letsencrypt/client/tests/client_test.py

```diff
@@ -0,0 +1,88 @@
+"""letsencrypt.client.client.py tests."""
+import unittest
+
+import mock
+import zope.component
+
+
+class RollbackTest(unittest.TestCase):
+    """Test the rollback function."""
+    def setUp(self):
+        self.m_install = mock.MagicMock()
+        self.m_input = mock.MagicMock()
+        zope.component.getUtility = self.m_input
+
+    def _call(self, checkpoints):  # pylint: disable=no-self-use
```

Comment:
To stay consistent with my recent changes at 23cab4e6943ff1a61f985dd67d2ebcf115ca9775 instead of `pylint: disable=no-self-use` you should use

```
@classmethod
def _call(cls, checkpoints):
    ...
```

Here, and in other `_call`s.